### PR TITLE
Feature/patro23-240-setup-azurite-container

### PR DESCRIPTION
**Pull request by AlanP22**
https://tracker.intive.com/jira/browse/PATRO23-240

### CODE REVIEW

File: .idea/.idea.Intive.Patronage2023/.idea/.gitignore
Comment (pzajaczkowski) [inline .idea/.idea.Intive.Patronage2023/.idea/.gitignore:1]: None of this files from .idea folder should be commited and pushed. Check if git ignores this folder if not maybe add directory to .idea.

File: .idea/.idea.Intive.Patronage2023/.idea/encodings.xml
Comment (pzajaczkowski) [inline .idea/.idea.Intive.Patronage2023/.idea/encodings.xml:1]: remove

File: .idea/.idea.Intive.Patronage2023/.idea/indexLayout.xml
Comment (pzajaczkowski) [inline .idea/.idea.Intive.Patronage2023/.idea/indexLayout.xml:1]: remove

File: .idea/.idea.Intive.Patronage2023/.idea/vcs.xml
Comment (pzajaczkowski) [inline .idea/.idea.Intive.Patronage2023/.idea/vcs.xml:1]: remove

File: docker-compose.override.yml
Comment (pzajaczkowski) [inline docker-compose.override.yml:1]: Why is there whole file marked as changed? Maybe your file encoding is different or sth.

**Review by pzajaczkowski — CHANGES_REQUESTED** [review]
If it works that's good. Just remove your ide files and check encodings or other reasons why your changes marked whole files as changed.

File: docker-compose.override.yml [hunk 1]
```diff
@@ -1,50 +1,59 @@
...
+      - DB_PASSWORD=S3cur3P@ssW0rd!
+      - JAVA_OPTS=-Xms512m -Xmx2048m
+    ports:
+      - 8080:8080
+      - 8443:8443
+    volumes:
+      - ./realm-config.json:/root/data/import/realm-config.json
+    restart: always
+  azurite:
+    image: mcr.microsoft.com/azure-storage/azurite
```
Comment (DawidBski) [inline docker-compose.override.yml:52]: I think it shouldn't be in docker-compose.override.yml especially if you put it in docker-compose.yml
image

**Review by DawidBski — CHANGES_REQUESTED** [review]

File: docker-compose.override.yml (hunk 1 above)
Comment (mateuszgiedzinski) [reply docker-compose.override.yml:52]: indeed

File: .idea/.idea.Intive.Patronage2023/.idea/vcs.xml
Comment (mateuszgiedzinski) [reply .idea/.idea.Intive.Patronage2023/.idea/vcs.xml:1]: please add .idea to gitignore file as in this gitignore
https://github.com/intive/patronage22-dotnet-bialystok/blob/master/.gitignore

Comment (mateuszgiedzinski) [reply .idea/.idea.Intive.Patronage2023/.idea/vcs.xml:1]: But before you need remove whole idea directory.

**Review by pzajaczkowski — APPROVED** [review]

File: docker/docker-compose.override.yml [hunk 2]
```diff
@@ -1,53 +1,65 @@
...
+    restart: always
+  smtp4dev:
+    ports:
+      - '3000:80' # WebInterface
+      - '25:25' # SMTP server
+      - '143:143' # IMAP server
+    environment:
+      - ServerOptions__HostName=smtp4dev
+  azurite:
+    container_name: "azurite"
```
Comment (mateuszgiedzinski) [inline docker/docker-compose.override.yml:59]: I think we put container name in docker-compose.yml. Here we have only environment specific config.  So everything is fine except container_name.

**Review by mateuszgiedzinski — CHANGES_REQUESTED** [review]

**Review by mateuszgiedzinski — APPROVED** [review]

File: docker/docker-compose.override.yml [hunk 3]
```diff
@@ -1,53 +1,64 @@
...
+    environment:
+      - ASPNETCORE_ENVIRONMENT=Development
+      - ASPNETCORE_URLS=https://+:443;http://+:80
+      - ASPNETCORE_Kestrel__Certificates__Default__Password=awesomepass
+      - ASPNETCORE_Kestrel__Certificates__Default__Path=/https/intivepatronageapi.pfx
+    ports:
+      - "5080:80"
+      - "5081:443"
+    volumes:
+      - ${HOME}/.microsoft/usersecrets:/root/.microsoft/usersecrets:ro
```
Comment (mateuszgiedzinski) [inline docker/docker-compose.override.yml:23]: fix paths

**Review by mateuszgiedzinski — CHANGES_REQUESTED** [review]

**Review by DawidBski — APPROVED** [review]

File: docker/docker-compose.override.yml (hunk 3 above)
Comment (AlanP22) [reply docker/docker-compose.override.yml]: done

**Review by mateuszgiedzinski — APPROVED** [review]

**Pull request merged**